scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -12,7 +12,7 @@ workflow GvsExtractCallset {
         File reference_index
         File reference_dict
 
-        String fq_cohort_extract_table_prefix
+        String cohort_extract_table_prefix
```

Comment:
does this have to change to be not fully qualified? I think it breaks for our use case since our cohort extract table does not reside at `"~{data_project}.~{default_dataset}.~{samples_to_extract_table}"` which is the default being set in L74 and L76

Reply:
Are you are using a different project for your temp tables? If so, how do you run the prepare callset wdl? Maybe you have already sent me this, but do you have a doc that describes what projects you are using?

Reply:
We don't have a great doc but this is the one that goes over the setup https://github.com/all-of-us/workbench/blob/master/api/genomics/docs/wgs_cohort_extraction_setup.md.

So the temp_tables_dataset and destination_dataset are currently in the same project but our WDL has the temp_table_dataset and destination_dataset being passed in as fq parameters so it could break if we move that around. https://github.com/broadinstitute/gatk/blob/ah_var_store/scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl#L17

The more problematic thing is that those temporary datasets and the gvs_dataset have different permissions which is why we need to specify the `query_project` parameter to tell Google which project credentials to use when executing a query. This is more relevant in the timestamp task where we're setting the query project through the `--project_id` parameter and specifying the parent project containing the dataset by passing in a fq table name `{project}:{dataset}.{table}`.